### DISCUSSION TITLE
Does flight risk in the calculator account for airport risk?

### DISCUSSION THREAD

**Discussion by josiah-roberts** [microCOVID/microCOVID#1118] [Q&A] [answered]
Given the filtering, lack of talking, and lack of mixing, risk on airplanes is comparatively low.

I am wondering, however, if the "Plane Ride" pre-made scenario on the calculator accounts for airport risk:
- Checking in
- Standing in line in security
- Lack of air filtration at the gate
- Etc.

Does that calculator account for this risk, or is it only modeling risk while _on the plane_?

**Comment by blanchardjeremy** [comment, marked as answer]
Thanks for asking! The answer to your question is: no, it just models the time on the plane. You can model a separate activity in the calculator and add the two scores together. 

Here is how I modeled a recent airport visit: https://www.microcovid.org/?distance=sixFt&duration=30&interaction=oneTime&personCount=5&riskProfile=average&scenarioName=custom&setting=indoor&subLocation=US_06001&theirMask=basic&topLocation=US_06&voice=normal&yourMask=basic

Also, I opened an issue over at #1122 for this to be included in a future version.
